RFR-App/RFR/RFRApp.swift

```diff
@@ -32,6 +33,18 @@ struct RFRApp: App {
     /// The application, which is required to store and load the authentication state of this application.
     @StateObject var appModel = AppModel()
 
+    init() {
+        SentrySDK.start { options in
+            options.dsn = "https://cfb4e7e71da45d9d7fc312d2d350c951@o4506585719439360.ingest.sentry.io/4506585723437056"
+            options.debug = false // Enabled debug when first installing is always helpful
+
+            // Enable tracing to capture 100% of transactions for performance monitoring.
+            // Use 'options.tracesSampleRate' to set the sampling rate.
+            // We recommend setting a sample rate in production.
+            options.enableTracing = true
```

Comment:
Shouldn't you follow the recommendation to set the sample rate, when you enable tracing?

In Android we don't use tracing (performance) yet, but when you want to use this here, make sure it does not take up too much resources in the app. You could also only enable tracing in you own builds (not for all) or only for some users (random-picks).

Also: In Android we only add sentry in the UI-library, as this requires us to ask the user if he wants to opt-in into Sentry (and show privacy policy notes for sentry). Are you sure you want to add this in the backend? In case you need it "down here" in this layer - maybe add a parameter to the API of the backend library so that the UI can decide if it should be enabled (and maybe even allow to add different Sentry DSN (Sentry user ids).

Reply:
This is the UI for RFR, not the backend.

I also added a setting to only trace in Development and Staging.